src/time_travel/time_machine_clock.py

```diff
@@ -0,0 +1,33 @@
+"""A single clock for all patchers."""
+
+
+class TimeMachineClock(object):
```

Comment:
Separated between time and timstamp.
Don't see any reason why float isn't good enough and to prove it I removed the advance_timestamp method.